test/remote/gateways/remote_d_local_test.rb

```diff
@@ -84,6 +85,15 @@ def test_successful_purchase_with_more_options
     assert_match 'The payment was paid', response.message
   end
 
+  def test_successful_purchase_with_additional_data
+    options = @options.merge(
+      additional_data: { item: 'socks' }
```

Comment:
After reviewing the DLocal docs, it seems that the `additional_risk_data` object then contains other objects rather than independent fields like `item` so this test example may be a bit misleading. I'd suggest we change it to match the gateway's expectations a bit more literally. I edited it locally to the following and it still passed so that's good!
` options = @options.merge(
      additional_data: {
        submerchant: {
          name: 'socks'
        }
      }
    )`